lib/services/in-memory-db.service.ts

```diff
@@ -110,13 +123,40 @@ export class InMemoryDBService<T extends InMemoryDBEntity> {
   }
 
   /**
-   * Get a single record of type `T` with the supplid id value.
+   * Remove the records of type `T` from the in-memory data store using the supplied PK ids.
+   * @param ids the PK ids of the records
+   */
+  public deleteMany(ids: number[]): void {
+    ids.forEach(id => {
+      const { [id]: removed, ...remainder } = this.recordMap;
+      this.recordMap = {
+        ...remainder,
+      };
+    });
```

Comment:
This is a bit interesting. It works, but I wonder the implications of it and if there is a different way to do it.